chain/bitcoind_conn.go

```diff
@@ -254,9 +297,15 @@ func (c *BitcoindConn) txEventHandler() {
 		default:
 		}
 
+		// Set up the buffers we expect our messages to consume. ZMQ
+		// messages from bitcoind include three parts: the command, the
+		// data, and the sequence number.
+		var command [len(rawTxZMQCommand)]byte
+		var seqNum [seqNumLen]byte
```

Comment:
lift these up to where `data` is initialized? can all use a single var statement, same for blocks

Reply:
Done.